README.md

```diff
@@ -73,19 +63,20 @@ vendor
 ```
 
 ## Stylelint
-We use `stylelint` to ensure a consistent style. See `/src/helpers/_styleguide-example.scss`
-for an example file that adheres to the `/.stylelintrc` rules. To lint manually run
-`npm run lint`, this task is also run before each commit as a pre-commit hook.
+We use `stylelint` to ensure a consistent style. See
+`/src/helpers/_stylelint-example.scss` for an example file that adheres to the
+`/.stylelintrc` rules. To lint manually run `npm run lint`, this task is also
+run before each commit as a pre-commit hook.
 
 ## Gulp
 To compile the SCSS run `npm install && gulp`. This is handy to ensure you don't
```

Comment:
Change to `npm install && node_modules/.bin/gulp` and/or add `npm` script `dev`?

Reply:
Changed to `npm install && node_modules/.bin/gulp`